GitUI/Hotkey/HotkeySettingsManager.cs

```diff
@@ -261,6 +261,8 @@ public static HotkeySettings[] CreateDefaultSettings()
                     hk(FormBrowse.Commands.QuickFetch, Keys.Control | Keys.Shift | Keys.Down),
                     hk(FormBrowse.Commands.QuickPull, Keys.Control | Keys.Shift | Keys.P),
                     hk(FormBrowse.Commands.QuickPush, Keys.Control | Keys.Shift | Keys.Up),
+                    hk(FormBrowse.Commands.Stash, Keys.Control | Keys.Alt | Keys.Up),
```

Comment:
These shortcuts are already used to navigate between commits when the focus is on the revision grid. Can't we find something else?

Reply:
Could you show where they are defined? I couldn't find any other shortcuts using "Keys.Control | Keys.Alt"

Reply:
Sorry, I didn't see the ctrl. A review on my phone :-(

Reply:
Ah, I see, it's OK ;)

Reply:
On a Mac ctrl is next to alt, and the new shortcuts may lead to a confusing or frustrating experience for a user with large or clumsy fingers...

Can we use some other shortcuts? May be something like a combo - "ctrl+s, ctrl+ ⬆️"?  (I don't know if it is possible)

Reply:
I don't think that's possible with current implementation. I am not sure that "clumsy fingers" is a valid argument :) Otherwise we would only have shortcuts with keys very very far away from each other. And if there would be such users (I doubt it) they could change these key bindings in the settings.

Reply:
Can you add `ShortcutKeyDisplayString` for these menu items to spread the knowledge?
Here is the valid place and an example:
https://github.com/gitextensions/gitextensions/blob/f5f8c8f260c78bb774cf585396c21ef43e375b89/GitUI/CommandsDialogs/FormBrowse.cs#L675

Reply:
Great idea, thanks! Made the changes and rebased on latest master.